library/validation/validation_script.js.php

```diff
@@ -213,6 +213,34 @@ function hideErrors(input, id){
                     $('a#header_tab_'+type_tab).css('color', 'black');
                 }
             }
+            /*
+            * Check if exist translation for current error message else return default message.
+            * In addition you can adding custom error message to the constraints json according validate.js instructions and add the translation here
+            * @param message
+            **/
+            function getErrorMessage(message){
+                //enable to translate error message
+                //todo - adding all the translations string from validate.js
+                // console.log(message);
+                switch (message){
+                    case 'Patient Name Required':
+                        return '<?php echo xlt('Patient Name Required');?>';
+                    case 'An end date later than the start date is required for repeated events!':
+                       return '<?php echo xlt('An end date later than the start date is required for repeated events!');?>';
+                    case 'Required field missing: Please enter the User Name':
+                        return '<?php xlt('Required field missing: Please enter the User Name','e');?>';
+                    case 'Please enter the password':
+                        return '<?php echo xlt('Please enter the password'); ?>';
+                    case 'Required field missing: Please enter the First name':
+                        return '<?php xlt('Required field missing: Please enter the First name');?>';
+                    case 'Required field missing: Please enter the Last name':
+                        return '<?php xlt('Required field missing: Please enter the Last name');?>';
+                    default:
+                       return '<?php echo xl('is not valid');?>';
```

Comment:
xla()

And since this is in javascript, recommend using xla() rather than xlt() for the other above items.